### PR TITLE
Added question in FAQ section

### PR DESCRIPTION
**Pull request by akgoel23**
Added "What other sample plug-ins are available?" question in FAQ section as we published a new sample plug-in.

### CODE REVIEW

File: WindowsServerDocs/identity/ad-fs/development/ad-fs-risk-assessment-model.md [hunk 1]
```diff
@@ -312,3 +312,9 @@ The method returns the [Risk Score](https://docs.microsoft.com/dotnet/api/micros
 
 **Why can't AD FS suggest the list of risky IPs, users, etc?**</br>
 **A:** Though not currently available, we are working on building the intelligence to suggest risky IPs, users, etc. in the Pluggable Risk Assessment Model. We will share the launch dates soon. 
+
+**What other sample plug-ins are available?**</br>
+**A:** Following sample plug-in(s) are available.
```
Comment (illfated) [inline WindowsServerDocs/identity/ad-fs/development/ad-fs-risk-assessment-model.md:317]:
```suggestion
**A:** The following sample plug-in(s) are available:
```

**Comment by illfated** [comment]
You may want to check with the page author that the GitHub link is preferable.

**Comment by eross-msft** [comment]
@billmath  Please review and let me know if it's okay to merge. Thanks!

**Comment by billmath** [comment]
#sign-off

**Pull request merged**